.gitignore

```diff
@@ -0,0 +1,11 @@
+build/*
+!build/
```

Comment:
If I am not wrong the exclamation mark would cause the build folder not to be ignored. Is there a specific reason why you are using it?